.github/workflows/kind.yml

```diff
@@ -215,7 +215,13 @@ jobs:
           mkdir log
           mkdir test-e2e-encap-all-features-enabled-coverage
           # FlowExporter requires the FlowAggregator, so we keep it disabled.
-          ANTREA_LOG_DIR=$PWD/log ANTREA_COV_DIR=$PWD/test-e2e-encap-all-features-enabled-coverage ./ci/kind/test-e2e-kind.sh --encap-mode encap --coverage --feature-gates AllAlpha=true,AllBeta=true,FlowExporter=false --proxy-all --multicast
+          ANTREA_LOG_DIR=$PWD/log ANTREA_COV_DIR=$PWD/test-e2e-encap-all-features-enabled-coverage ./ci/kind/test-e2e-kind.sh \
+            --encap-mode encap \
+            --coverage \
+            --feature-gates AllAlpha=true,AllBeta=true,FlowExporter=false \
```

Comment:
nit: we can remove `FlowExporter=false` now that https://github.com/antrea-io/antrea/pull/5021 has been merged
(and remove the comment above)

Reply:
done